src/main/java/betterquesting/questing/tasks/TaskFluid.java

```diff
@@ -45,16 +57,23 @@ public class TaskFluid implements ITaskInventory, IFluidTask, IItemTask {
     public boolean consume = true;
     public boolean groupDetect = false;
     public boolean autoConsume = false;
+    private boolean fold = true; // This remains through the game.
```

Comment:
this is for the compact view right? In that case:

Shouldnt this false by default and be stored in a config to retrieve next time?

'false' because most people dont play on tiny 640x480 screens that need a compact view.